_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,8 @@
+
+The last file required to make a Foreman cluster work is __$app_root/tmp/secret_token.rb__. This file should contain a secret token that is modeled after __$app_root/config/initializers/local\_secret\_token.rb__. __$app_root/tmp/secret_token.rb__ is used by __$app_root/config/initializers/secret_token.rb__ to sign cookies. Once you have created the new secret_token.rb, restart Foreman and clear Foreman's cache:
+{% highlight bash %}
+foreman-rake tmp:cache:clear
+foreman-rake tmp:sessions:clear
+{% endhighlight %}
+
+Note: The file __$app_root/config/initializers/secret_token.rb__ is uncommented, the logic behind this step rests on suggestions from IRC and testing. Without this change, the user may need to log in multiple times, may run in to "Invalid Authenticity Token"/CSRF issues, or it might just work for no clear reason. 
```

Comment:
I don't think these comments about whether it may or may not work, or how you heard about it should be in the manual. The info about the symptom may be useful though.